server/controllers/admin/suppliers.js

```diff
@@ -66,26 +78,50 @@ function detail(req, res, next) {
   .done();
 }
 
-// POST /supplier
+/**
+ * @method create
+ *
+ * @description
+ * POST /supplier
+ *
+ * This method creates a new supplier entity in the database and sets up the
+ * creditor for the it.
+ */
 function create(req, res, next) {
   const data = db.convert(req.body, ['creditor_uuid']);
 
   // provide uuid if the client has not specified
-  data.uuid = db.bid(data.uuid || uuid());
+  const uid = data.uuid || uuid();
```

Comment:
Briefly reading the `node-uuid` source code shows that `uuid.v4` is assigned as the library export - this means both provide the same output but it may still be a good idea to stick to one or another for the sake of future developers.